_data/projects.yml

```diff
@@ -58,12 +58,12 @@
     - news
     - health
 
-- logo: employ-ohio-now.svg
-  link: #
-  title: Employ Ohio Now
-  description: Find jobs and fill talent needs created by COVID-19
-  tags:
-    - employment
+# - logo: employ-ohio-now.svg
```

Comment:
just delete them instead of commenting it out. we can always back out the commit if we need to, or put the project back into the code.